java/java.lsp.server/vscode/README.md

```diff
@@ -52,6 +52,11 @@ Apache NetBeans Language Server brings full featured Java development (edit-comp
 * __Test Explorer__ for Java tests results visualization and execution including editor code Lenses.
 * Maven and Gradle support including multi-project projects, subprojects opening and Gradle priming builds.
 
+## Project Explorer
+Project Explorer is introduced, as a counterpart for File Explorer. The Project Explorer follows logical project's structure, groups sources together and in case of Java, simplifies package structure view. In the future, Project Explorer will offer build, execute and build system (gradle/maven) oriented 
```

Comment:
Isn't "counterpart" negative word? "is introduced" belongs to change log, not to readme, I think. The build action & co. have been added meanwhile, haven't they? I am not sure whether people know what _File Explorer_ is? The view is labeled _workspace_ if I have more than one folder open...

_Project Explorer provides an overview of logical project structure, groups sources together and greatly simplifies Java package structure exploration. Project Explorer is an excellent addition to the classical workspace explorer. Use it to build, test, execute and operate your Maven and Gradle Java projects._

![obrazek](https://user-images.githubusercontent.com/26887752/148724405-cdbff0c7-1040-4305-bf47-8da03ccce2ed.png)

Reply:
Fixed